tests/data/kern/Bach-BWV_316.krn

```diff
@@ -0,0 +1,116 @@
+!!!COM: Bach, Johann Sebastian
```

Comment:
@melkisedeath: Do we have the right to add this file to the repo? I assume their license allows that, but probably we need to add an acknowledgement somewhere in the README/Documentation...

Reply:
Hmm good question. There are a couple of royalty free Kern scores out there we could use instead or I could just take or write segments for the tests. Definitely, the tests still need some work but I think I'm gonna add tests as the use cases and exceptions occur. So for the moment just let it be and I will edit the tests and test files as we go.